script/node-disabled-tests.json

```diff
@@ -295,6 +296,7 @@
   "parallel/test-vm-module-basic",
   "parallel/test-repl-sigint-nested-eval",
   "parallel/test-repl-sigint",
+  "parallel/test-util-inspect",
```

Comment:
why are these tests failing..? is it something we should be concerned about?

Reply:
this one specifically needs to be disabled to this new test:
```
{
  const ab = new ArrayBuffer(42);
  assert.strictEqual(ab.byteLength, 42);
  new MessageChannel().port1.postMessage(ab, [ ab ]);
  assert.strictEqual(ab.byteLength, 0);
  assert.strictEqual(util.inspect(ab),
                     'ArrayBuffer { (detached), byteLength: 0 }');
}
```
investigation turned up that this was due to v8 differences, the `ArrayBuffer` does not throw as detached when run with Electron.